style.css

```diff
@@ -0,0 +1,193 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'Open Sans', sans-serif;
+  font-family: 'Poppins', sans-serif;
+  font-family: 'Roboto', sans-serif;
+  font-family: 'Space Mono', monospace;
+}
+
+/* MOBILE VERSION */
+@media only screen and (min-width: 375px) {
+  #nav-bar {
+    display: flex;
+    width: 80%;
+    justify-content: space-between;
+    margin: auto;
+    align-items: center;
+    background-image: url(./Images/header-background-mobile.png);
+    position: sticky;
+    top: 0;
+    z-index: 4;
+    padding-top: 20px;
+  }
+
+  #side-navbar-container {
+    display: none;
+  }
+
+  #union-img:hover {
+    opacity: 0.8;
+  }
+
+  #union-img:active {
+    opacity: 3;
+  }
+
+  #logo-image {
+    width: 5rem;
+    height: 1.25rem;
+  }
+
+  #vector-image {
+    border: 2px solid rgb(92, 54, 166);
+    width: auto;
+    height: 30px;
+  }
+
+  main {
+    height: 100vh;
+    background-image: url(./Images/header-shapes\ mobile.png);
+    background-size: cover;
+  }
+
+  #main-container {
+    display: flex;
+    flex-direction: column;
+    width: 70%;
+    margin: auto;
+    padding-top: 100px;
+  }
+
+  /* MAIN SECTION START  */
+  #main-header {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 2.5rem;
+    line-height: 130%;
+    display: flex;
+    color: #172b4d;
+    width: 20rem;
+    height: 10rem;
+    align-items: center;
+    margin: 4rem 1.5rem 0.75rem;
+  }
+
+  #link {
+    font-family: 'Poppins', sans-serif;
+    color: #7f8cff;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1rem;
+    line-height: 150%;
+    margin-left: 1.5rem;
```

Comment:
- Let's remove this margin-left to make it be aligned at the left on its parent container since the section has aligned items of flex-start.